pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -150,7 +150,7 @@ const (
 // AzVolumeAttachmentStatus is the status for a AzVolumeAttachment resource
 type AzVolumeAttachmentStatus struct {
 	Role           Role              `json:"role"`
-	PublishContext map[string]string `json:"publish_context"`
+	PublishContext map[string]string `json:"publish_context,omitempty"`
```

Comment:
Need to run `./hack/update-crd.sh` and `./hack/update-codegen.sh` to update the CRD YAMLs for generated code. Then the changes to the YAMLs need to be copied to the appropriate files under `deploy\v2.0.0-alpha.1` and `charts\v2.0.0-alpha.1\azuredisk-csi-driver.` and helm package and index updated:

```console
$ helm package charts/v2.0.0-alpha.1/azuredisk-csi-driver -d charts/v2.0.0-alpha.1/
$ helm repo index charts --url=https://raw.githubusercontent.com/abhisheksinghbaghel/azuredisk-csi-driver/master/charts```